package.json

```diff
@@ -58,15 +59,16 @@
     "@opentelemetry/api-metrics": "0.23.0",
     "@opentelemetry/instrumentation-express": "0.32.1",
     "@opentelemetry/instrumentation-http": "0.35.1",
-    "compression": "^1.7.4",
     "@turf/area": "^6.5.0",
     "@turf/bbox-polygon": "^6.5.0",
     "@turf/ellipse": "^6.5.0",
     "@turf/intersect": "^6.5.0",
     "@turf/union": "^6.5.0",
     "@types/geojson": "^7946.0.8",
     "ajv": "^8.12.0",
+    "aws-sdk": "^2.1555.0",
     "axios": "^0.27.2",
+    "compression": "^1.7.4",
```

Comment:
Why did you move lines?
Refactor this file so I will see minimum changes

Reply:
refactored